bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1126,16 +1127,16 @@ void BCStateTran::trySendFetchBlocksMsg(uint64_t firstRequiredBlock,
   metrics_.last_msg_seq_num_.Get().Set(lastMsgSeqNum_);
 
   msg.msgSeqNum = lastMsgSeqNum_;
-  msg.firstRequiredBlock = firstRequiredBlock;
-  msg.lastRequiredBlock = lastRequiredBlock;
+  msg.minBlockId = minBlockId;
+  msg.maxBlockId = maxBlockId;
   msg.lastKnownChunkInLastRequiredBlock = lastKnownChunkInLastRequiredBlock;
 
   LOG_DEBUG(logger_,
             "Sending FetchBlocksMsg:" << reason
                                       << KVLOG(sourceSelector_.currentReplica(),
                                                msg.msgSeqNum,
-                                               msg.firstRequiredBlock,
-                                               msg.lastRequiredBlock,
+                                               msg.minBlockId,
+                                               msg.maxBlockId,
```

Comment:
lastKnownChunkInLastRequiredBlock doesn't look good with newly renamed maxBlockId from lastRequiredBlock.

Reply:
The name is really bad but it is unrelated to the change of min/max. I think we should change it later, not now.
As I said, there are many other things to refactor.
All the naming here is problematic..
I changed minimal things.